hadoop-ozone/dist/src/main/smoketest/security/ozone-secure-recon.robot

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Smoke test to start cluster with docker-compose environments.
+Library             OperatingSystem
+Library             String
+Library             BuiltIn
+Resource            ../commonlib.robot
+
+*** Variables ***
+${ENDPOINT_URL}     http://recon:9888
+
+*** Test Cases ***
+Recon REST API
+    Run Keyword if      '${SECURITY_ENABLED}' == 'true'     Kinit HTTP user
+    ${result} =         Execute                             curl --negotiate -u : -v ${ENDPOINT_URL}/api/containers
+                        Should contain      ${result}       containers
```

Comment:
The robot tests look good, but I think we could get rid of the duplication.  `security/ozone-secure-recon.robot` works in both secure and unsecure env.  So I propose:

1. rename `security/ozone-secure-recon.robot` to `recon/recon-api.robot` (ie. get rid of the unsecure-only test, but keep its name and location)
2. add `execute_robot_test recon recon` in `ozonesecure/test.sh`, too

This way the same test would be executed in both secure and unsecure env, similar to eg. `s3` tests:

https://github.com/apache/hadoop-ozone/blob/1caf1e30865aa2b380a7fca6d87f5ae8034fee4e/hadoop-ozone/dist/src/main/compose/ozone/test.sh#L39

https://github.com/apache/hadoop-ozone/blob/1caf1e30865aa2b380a7fca6d87f5ae8034fee4e/hadoop-ozone/dist/src/main/compose/ozonesecure/test.sh#L36

Reply:
@adoroszlai Thank you for the suggestion. I have addressed it.